src/bosh-softlayer-cpi/softlayer/pool/softlayer_pool_creator.go

```diff
@@ -107,46 +110,49 @@ func (c *softLayerPoolCreator) createFromVMPool(agentID string, stemcell bslcste
 			if err != nil {
```

Comment:
I don't understand this `if err != nil` check since lines 95 to 97 already deal with any error. Where would `err` change after line 97 and before 110?